core/src/mindustry/core/UI.java

```diff
@@ -629,6 +632,58 @@ public void showMenu(String title, String message, String[][] options, Intc call
         }}.show();
     }
 
+    /** Shows a menu that hides when another followUp-menu is shown or when nothing is selected.
+     * @see UI#showMenu(String, String, String[][], Intc) */
+    public void showFollowUpMenu(int menuId, String title, String message, String[][] options, Intc callback) {
+        Dialog dialog = new Dialog(title){{
```

Comment:
In terms of constructing a dialog, isn't this code identical to that of a standard menu? Looks like a lot of code duplication to me, this should be extracted to a separate method that makes a dialog if possible.

Reply:
Yes I thought so, but didn't want to modify things too much, will fix it asap